sdk/textanalytics/Azure.AI.TextAnalytics/api/Azure.AI.TextAnalytics.netstandard2.0.cs

```diff
@@ -1,9 +1,15 @@
 namespace Azure.AI.TextAnalytics
 {
+    [System.FlagsAttribute]
+    public enum AdditionalSentimentAnalyses
```

Comment:
`Additional` doesn't feel quite right when I say it to myself.  I'd suggest we discuss using a name like `SentimentAnalysisTraits` to match the `Traits`/`States` pattern we picked for Storage _(and mostly to revisit that discussion since nobody liked the suffix `Traits`, but it was the least hated from the statistically valid poll I took of the team room that day)_.  This should happen in ApiView rather than block the PR though.

Reply:
Thanks Ted.  I will start the discussion in the API view. 
Merging the PR for now

Reply:
Comment added to revision: https://apiview.dev/Assemblies/Review/b196a5e53fa2441f92e3b1accefa35e7